PolyEngine/Engine/Src/CoreConfig.hpp

```diff
@@ -11,6 +22,19 @@ namespace Poly
 		bool DebugNormalsFlag = false;
 		bool WireframeRendering = false;
 		bool DisplayFPS = true;
+
+		Dynarray<String> GetAssetsPaths(eResourceSource Source) const;
+
+		bool GetIsLoadedFromFile() const;
```

Comment:
ok, it was workaround for property named the same. What is convention in such situation?